js/background.js

```diff
@@ -3,6 +3,7 @@ const _browser = this.browser || this.chrome;
 const storage = _browser.storage.sync || _browser.storage.local;
 
 var openTicket = (ticket, newTab) => {
+  ticket = ticket.replace(/%20/gi, '');
```

Comment:
Since we are replacing some characters anyways, it might be better to use `/\s/g` instead. This does replace all whitespace characters (not just spaces - tabs, linefeeds etc.), too